build-and-deploy.sh

```diff
@@ -108,10 +108,25 @@ maybeDeploySmokeTestsQA() {
   fi
 }
 
+dispatch_github_event() {
```

Comment:
I'd just change the name to camelCase, since is used for the other function.

Reply:
Yep sorry. I will